tests/utilities.lisp

```diff
@@ -113,8 +117,24 @@ Returns (values SOURCE-PATHNAME COMPILED-PATHNAME)."
                      (check-string= (format nil "program text.~%~
 input file: ~A~%~
 line number: ~A~%~
-test case: ~A~%~
+test number: ~A~%~
+test header: ~A~%~
 expected error (A) and generated error (B)"
-                                            file line description)
+                                            file line
+                                            (or number "(unassigned)")
+                                            description)
                                     expected-error
                                     generated-error))))))
+
+(defun run-single (pathname test-number)
```

Comment:
I think this could just be `run-test` unless that name is already taken.

Reply:
done